github/Requester.py

```diff
@@ -538,6 +538,7 @@ def __makeAbsoluteUrl(self, url):
                 self.__hostname,
                 "uploads.github.com",
                 "status.github.com",
+                "github.com",
```

Comment:
I am a tiny bit wary about this. But I also can't see any other way to deal with it.